diff -urN giFT-0.11.1.orig/src/mime.c giFT-0.11.1/src/mime.c
--- giFT-0.11.1.orig/src/mime.c	2003-07-09 08:42:53.000000000 +0200
+++ giFT-0.11.1/src/mime.c	2003-07-26 21:30:20.000000000 +0200
@@ -25,7 +25,7 @@
 
 #ifdef USE_LIBMAGIC
 # include <magic.h>
-# define MIME_MAGIC_FILE "/usr/share/misc/file/magic.mime"
+# define MIME_MAGIC_FILE "/usr/share/file/magic.mime"
 #endif /* USE_LIBMAGIC */
 
 /*****************************************************************************/
