diff -Nru gift-0.11.4.orig/lib/platform.c gift-0.11.4/lib/platform.c
--- gift-0.11.4.orig/lib/platform.c	2003-08-16 13:07:48.000000000 +0200
+++ gift-0.11.4/lib/platform.c	2003-11-11 15:44:49.000000000 +0100
@@ -163,6 +163,8 @@
 static BOOL unix_init (const char *home, const char *local,
                        const char *data, const char *plugin)
 {
+	char *home_etc = getenv("HOME_ETC");
+	char path[1024];
 	/*
 	 * Attach signal handlers.  At some point in the future it would be nice
 	 * to check for the precense of signal and its quirks before using it...
@@ -188,7 +190,9 @@
 		return FALSE;
 	}
 
-	local_dir  = STRDUP ((local)  ? (local)  : (file_expand_path("~/.giFT")));
+	if (home_etc) snprintf(path, 1024, "%s/.giFT", home_etc);
+	
+	local_dir  = STRDUP ((local)  ? (local)  : (home_etc ? path : (file_expand_path("~/.giFT"))));
 
 	return TRUE;
 }
--- gift-0.11.4.orig/gift-setup.in	2003-08-12 06:25:17.000000000 +0200
+++ gift-0.11.4/gift-setup.in	2003-11-11 15:57:17.000000000 +0100
@@ -25,7 +25,7 @@
 ###############################################################################
 
 my $datadir = '@giftdatadir@';
-my $confdir = "$ENV{HOME}/.giFT";
+my $confdir = $ENV{HOME_ETC} ? "$ENV{HOME_ETC}/.giFT" : "$ENV{HOME}/.giFT";
 
 ###############################################################################
 
