--- giFT-0.11.1/configure.ac.orig	Sun Jul 13 01:48:32 2003
+++ giFT-0.11.1/configure.ac	Wed Jul 16 15:52:13 2003
@@ -337,9 +337,6 @@
 DL_LDFLAGS=
 DL_CFLAGS=
 
-# Override -g -O2...there must be a better way...
-CFLAGS="-g -Wall"
-
 dnl ###########################################################################
 dnl Architecture specific stuff...
 dnl ###########################################################################
